tests/installation.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
```

Comment:
Nice. I've never heard about BATS before, and it looks really interesting. :+1:

Reply:
I'm doing a pretty bad use here, because the tests build on each other, which is not what you're supposed to do. But the results are much easier to read than the results of a script that just terminates when something breaks.